index.bs

```diff
@@ -79,6 +79,12 @@ A <dfn>compression context</dfn> is the internal state maintained by a compressi
    * It is an error for DecompressionStream if the `ADLER32` checksum is not correct.
    * It is an error if there is additional input data after the `ADLER32` checksum.
 
+: `deflate-raw`
+:: "The DEFLATE algorithm" [[!RFC1951]]
+
+   * Implementations must be "compliant" as described in [[!RFC1951]] section 1.4.
+   * Field values described as invalid in [[!RFC1951]] must not be created by CompressionStream, and are errors for DecompressionStream.
```

Comment:
The format doesn't have "fields" as such. Maybe we should say "Non-conforming blocks"?

We need to define whether other data is permitted to follow a block with BFINAL set. I think we should find out what zlib does and specify that. https://datatracker.ietf.org/doc/html/rfc7692#section-7.2.1 implies that to implement WebSocket permessage-deflate correctly requires some special handling of BFINAL=1 blocks, but maybe that can be an option to add later.

Reply:
Done.

BFINAL is described in RFC1951.

```
         BFINAL is set if and only if this is the last block of the data
         set.
```

We cannot generate an error after closing a stream, so we ignore such blocks. Is it worth mentioning?

The special handling for permessage-deflate is needed because we tried to save a few bytes, which we don't do for this, IIUC.

Reply:
> We cannot generate an error after closing a stream, so we ignore such blocks. Is it worth mentioning?

Hmm, I was wrong. The decompression stream has two end-of-stream signals, one is the close() call and the other is BFINAL. We should fire an error when these two signals match with each other.